client/app/bundles/course/assessment/submission/reducers/liveFeedback.js

```diff
@@ -142,9 +142,18 @@ const liveFeedbackReducer = function (state = initialState, action) {
     case actions.LIVE_FEEDBACK_FAILURE: {
       const { questionId } = action.payload;
       return produce(state, (draft) => {
+        const previousConversation = getConversation(draft, questionId);
+        const errorMessage = {
+          text: ['An error occurred while processing your request !!!'],
```

Comment:
If you need this text, don't put it as hard-coded string over here. Use translations instead